testing/trino-tests/src/test/java/io/trino/execution/TestGrantOnTable.java

```diff
@@ -135,6 +135,7 @@ public void testGrantOnNonExistingTable(String privilege)
     {
         return new Object[][] {
                 {"SELECT"},
+                {"UPDATE"},
```

Comment:
Since you are updating this, can you please add `INSERT` and `DELETE` too?

Reply:
done